client/my-sites/upgrades/checkout/checkout.jsx

```diff
@@ -73,6 +72,7 @@ const Checkout = React.createClass( {
 		}
 
 		window.scrollTo( 0, 0 );
+		analytics.luckyOrange.initialize();
```

Comment:
fun fact! we have analytics middleware for just this kind of situation - we would strongly prefer to keep all analytics calls and side-effecting things out of the components themselves. let's consider augmenting the existing analytics middleware in `state/analytics` before we start injecting code like this into components :smile: